Tasks/VsTest/vstest.ts

```diff
@@ -963,4 +939,59 @@ function isNullOrWhitespace(input) {
         return true;
     }
     return input.replace(/\s/g, '').length < 1;
-}
\ No newline at end of file
+}
+
+class VSTestVersion {
```

Comment:
the things with approach is the the classes will keep on getting added for every version you need to support something.
Ideally a simple function method in js should resolve that and create anonymously. Simple and elegant.